.gitignore

```diff
@@ -21,6 +21,8 @@
 /*.egg-info
 /htmlcov
 /docs/_build
+/.venv
+/.vscode
```

Comment:
Please do not use this ignore file to define platform specific files.

For these purposes create a global .gitignore file, which is a list of rules  for ignoring files in every Git repository on your workstation.

See https://docs.github.com/en/get-started/getting-started-with-git/ignoring-files#configuring-ignored-files-for-all-repositories-on-your-computer for details